src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -113,6 +114,11 @@ private void OnDispatchedToUI(object sender, UIDispatcherEventArgs e)
         /// </summary>
         public event Action<PortModel> PortDisconnected;
 
+        /// <summary>
+        /// Event triggered when node is ready after update graph task.
+        /// </summary>
+        public event Action<bool> EvaluationEdged;
```

Comment:
Ok, It's bad name... I meant to say EvaluationStarted and EvaluationEnded and called them together - `EvaluationEdged`.